src/auth/oauth2client.ts

```diff
@@ -459,7 +459,7 @@ export class OAuth2Client extends AuthClient {
   }
 
   protected static readonly GOOGLE_TOKEN_INFO_URL =
-    'https://oauth2.googleapis.com/tokeninfo';
```

Comment:
At some point, I was asked to move *to* this endpoint.  Could I trouble you to start a thread with me, Chris Broadfoot, and Hong on this specific point internally?

Reply:
Yeah, sure thing. I think I was involved in that earlier thread. That is how I got familiar with this issue.

Reply:
After offline discussion, the old endpoint now supports POST requests. I have reverted to the old endpoint.